docs/UserGuide.adoc

```diff
@@ -127,22 +127,12 @@ Alias Example:
 
 * `a n/Ben p/12345678 e/ben@ten.com a/Unicorn Lane t/SuperHero`
 
-==== Listing all persons : `list`
+==== Listing all persons: `list`
 
-Shows a list of persons in the address book. +
+Shows a list of all persons in PIMP. +
 Format: `list`
 
-A list of all persons is shown.
-
-Examples:
-
-* `list` +
-Lists all persons.
-
-
-Alias example:
```

Comment:
Shouldn't be removing the alias example.

Reply:
I moved this to the bottom part to make it consistent with others